src/main/ipc-handlers.ts

```diff
@@ -108,12 +122,28 @@ export const initializeIpcHandlers = (): void => {
 						}
 					}
 				}
-			} else if (desktop_platform === 'linux') {
-				// TODO: Platform checking on Linux
-				// Set 'game_platform.available' true and setup data if platform is available, do nothing otherwise
-				continue;
 			}
+			return DefaultWindowsGamePlatforms;
+		} else if (desktop_platform === 'linux') {
+			for (const key in DefaultLinuxGamePlatforms) {
+				const game_platform = DefaultLinuxGamePlatforms[key];
+				if (game_platform.key === GamePlatform.STEAM) {
+					// TODO: Check and support other types of steam installations; Flatpak
+					try {
+						const buff = readFileSync(homedir() + '/.steam/registry.vdf');
+						const steamVdf = parse(buff.toString()) as {Registry:{HKCU:{Software:{Valve:{Steam:{Apps:object}}}}}};
+						//tries to find Among Us's Steam Id in the .vdf-file
+						if ("945360" in steamVdf["Registry"]["HKCU"]["Software"]["Valve"]["Steam"]["Apps"]) {
+							game_platform.available = true;
+						}
```

Comment:
Good job finding this, I definitely didn't know it existed in Linux and it's basically a mirror of the windows registry entry steam has - I should (and will in the new version thanks to you) be checking it in Windows for if Among Us is present instead of just blindly trying to run the URI.

Do you happen to know how cross-distro this is? Is it in EVERY distro, most distros, some, etc..?

Reply:
I am fairly certain all natively installed packages of Steam uses a registry.vdf file in this location. Thus should be working on any distro where Steam is not sandboxed like in Flatpak or ~Snap~(no Steam package there).

I've so far checked packages from Manjaro repo, Fedora repo, Ubuntu repo and the steam-installer.deb directly from Steam's website, and they all use this file in this location.